src/py_eddy_tracker/generic.py

```diff
@@ -76,6 +77,7 @@ def build_index(groups):
     >>> build_index(array((1, 1, 3, 4, 4)))
     (array([0, 2, 2, 3]), array([2, 2, 3, 5]), 1)
     """
+
```

Comment:
why new blank line

Reply:
Sometimes blanck lines appear after the docstring, sometimes not, I'm confused

Reply:
(in groups.py for example)